src/masonite/orm/grammar/BaseGrammar.py

```diff
@@ -102,6 +105,19 @@ def _compile_alter(self):
         self._sql = sql
```

Comment:
![image](https://user-images.githubusercontent.com/51742802/76617361-82f74200-64fc-11ea-9660-3f51fd61820f.png)
It's should be 
```MySQL
alter table article 
add constraint article_user_id_foreign foreign key(`user_id`) references `users`(`id`)
```

Reply:
I think you should update the alter to compile the  foreign constraints

Reply:
@corentinalcoy yes you are right.

Reply:
I'll need to refactor the alter grammar like I did for the create grammar